add_organism.py

```diff
@@ -0,0 +1,141 @@
+
+import sys
+import pprint 
+import os
+import pickle
+import subprocess
+import json
+from pprint import pprint
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'phageAPI.settings')
+import django
+django.setup()
+
+from restapi.models import (
+    Organism,
+    Spacer,
+    Repeat,
+    LocusSpacerRepeat,
+    AntiCRISPR,
+    Locus
+)
+
+### Constants ###
+Entrez_email = 'wenxinchen11@hotmail.com'
```

Comment:
I would make this a command-line argument, you personally don't want to be responsible for ALL the downloads :)

Reply:
Fixed :)